frontend/helpers/path-helpers.ts

```diff
@@ -22,6 +22,7 @@ type ProductListPathAttributes = Pick<
 
 export function productListPath(
    productList: ProductListPathAttributes,
+   itemType?: string,
    variant?: string | undefined
 ): string {
```

Comment:
Since we now have two arguments that are `string | undefined`, it might make sense to have this function accept an object instead of arguments to prevent mix ups

i.e.
```ts
function productListPath({ productList, itemType, variant}: {
   productList: ProductListPathAttributes,
   itemType?: string,
   variant?: string | undefined
}) { ... }
```

Reply:
Yea, I was wondering about that. I built this branch before that code was merged. I did the quick fix on merge, but I thought this might make sense as well

Opinions on doing it now vs next time we touch it?

Reply:
Next time